blog/_posts/2017-09-25-serv0101-conference-blog.md

```diff
@@ -0,0 +1,67 @@
+---
+title: JSFoo 2017 - Team Serv0101 
+layouts: post
+created_at: Mon Sep 25 2017
+permalink: blog/2017-09-25-Team-serv0101
+current: blog
+author: Neha & Rakhi
+categories:
+- student-posts
+---
+
+## Team Serv0101 at JSFoo 2017
+<img src="/img/blog/2017/2017-09-25-Serv0101-JSfoo2017.png">
+<div class="image-credits">Screenshot</div>
```

Comment:
Hi, thanks for teh PR. Can you please align the credits in the center for all images?